helpers/vampire.js

```diff
@@ -1,92 +1,130 @@
-const { execFile } = require('child_process');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
```

Comment:
Toto je síce super jednoduchý spôsob ako spromisifikovať funkciu s callbackom, ale aj veľmi neflexibilný.

Ak chcete využiť `execFile` a poslať vstup cez `stdin` (čo je bezpečnejšie ako `exec`), treba ju spromisifikovať manuálne podľa príkladu na https://javascript.info/promisify, napríklad:
```javascript
const { execFile } = require('child_process');
const execFileWithInput = (file, args, input, options, callback) =>
  new Promise((resolve, reject) => {
    const child = execFile(file, args, options,
                    (error, stdout, stderr) =>
                      error ? reject(error) : resolve({stdout, stderr}));
    child.stdin.write(input);
    child.stdin.end();
  })
```
Možno som spravil nejakú chybu, skúste to odladiť.